react/MonthPicker/MonthPicker.demo.js

```diff
@@ -61,6 +62,7 @@ export default {
         }
       ]
     },
-    ...fieldMessageOptions
+    ...fieldMessageOptions,
+    ...fieldLabelOptions
```

Comment:
Yeah I thought so but it doesn't work correctly currently as it has a custom `maxCharacters` prop. We should probably roll that back in favour of secondary label but don't think this is the PR for it.